src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -7484,6 +7484,91 @@ def test_aks_create_with_standard_sku(self, resource_group, resource_group_locat
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=79642318&view=results) to validate the change.

Reply:
test passed